### PR TITLE
module base pre alpha revamp

### PR DESCRIPTION
**Pull request by dhaavi**

### CODE REVIEW

File: modules/base/src/components/About.vue [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      <div v-if="$parent.versions">
+        <p style="padding: 5px;">
+          <strong>Name</strong>:         {{ $parent.versions.Core.Name }}<br/>
+          <strong>Version</strong>:      {{ $parent.versions.Core.Version }}<br/>
+          <strong>Commit</strong>:       {{ $parent.versions.Core.Commit }}<br/>
+          <strong>BuildOptions</strong>: {{ $parent.versions.Core.BuildOptions }}<br/>
+          <strong>BuildUser</strong>:    {{ $parent.versions.Core.BuildUser }}<br/>
+          <strong>BuildHost</strong>:    {{ $parent.versions.Core.BuildHost }}<br/>
+          <strong>BuildDate</strong>:    {{ $parent.versions.Core.BuildDate }}<br/>
+          <strong>BuildSource</strong>:  {{ $parent.versions.Core.BuildSource }}<br/>
```
Comment (ppacher) [inline modules/base/src/components/About.vue:42]: For better readability consider adding a space after each `Build` (i.e. `Build Options`)

File: modules/base/src/components/Base.vue [hunk 2]
```diff
@@ -125,73 +126,114 @@ export default {
...
+      });
     }
   },
   methods: {
-    selectUIModule(index) {
-      this.uiModules[index].loaded = true;
-      this.activeModule = this.uiModules[index].url;
+    selectUIModule(url) {
+      for (const index in this.uiModules) {
+        if (this.uiModules[index].url == url) {
```
Comment (ppacher) [inline modules/base/src/components/Base.vue:171]:
```suggestion
        if (this.uiModules[index].url === url) {
```

In JS, always use strict equality (`===`) unless you really want a loose check (`==`).

File: modules/base/src/components/Support.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+      <p>
+        You can ask for help on our <a href="https://www.reddit.com/r/safing">Subreddit r/safing</a>.
+      </p>
+
+      <h3>Report Bugs</h3>
+
+      <p>
+        Please report bugs in the User Interface in the <a href="https://github.com/safing/portmaster-ui/issues">safing/portmaster-ui Github repo</a>, and bugs in the Portmaster Core in the <a href="https://github.com/safing/portmaster/issues">safing/portmaster Github repo</a>.
+      </p>
```
Comment (ppacher) [inline modules/base/src/components/Support.vue:22]: Maybe mention the actual portmaster repository for firewalling/service bugs as well

Comment (dhaavi) [reply modules/base/src/components/Support.vue:22]: There is a second link ;)

File: modules/base/src/components/Base.vue (hunk 2 above)
Comment (dhaavi) [reply modules/base/src/components/Base.vue:171]: Ah. JS. 😏

File: modules/base/src/components/About.vue (hunk 1 above)
Comment (dhaavi) [reply modules/base/src/components/About.vue:42]: Yup. Good point.

File: modules/base/src/components/About.vue [hunk 4]
```diff
@@ -0,0 +1,111 @@
+<template>
+
+  <div class="about">
+    <div class="ui basic segment">
+
+      <h3>License</h3>
+
+      <p>
+        The Portmaster is licensed under the AGPLv3.<br />
+        More attribution to other Projects and their licenses coming soon.
```
[the remark this answers is not in the document]
Comment (dhaavi) [reply modules/base/src/components/About.vue:10]: Yup.

File: modules/base/src/components/About.vue [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+            <th>License</th>
+            <th>Notice</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>MaxMind GeoLite2</td>
+            <td>EULA with CC BY-SA 4.0</td>
+            <td>This product includes GeoLite2 data created by MaxMind, available from <a href="http://www.maxmind.com">http://www.maxmind.com</a>.</td>
+          </tr>
```
[the remark this answers is not in the document]
Comment (dhaavi) [reply modules/base/src/components/About.vue:27]: Well, we should delete old releases when there are new ones, but there are none. So, I think we'll be fine until we have a good solution to this.

File: modules/base/src/components/Support.vue (hunk 3 above)
Comment (ppacher) [reply modules/base/src/components/Support.vue:22]: Lol, I missed the scroll bar :D

**Pull request merged**